api/src/main/java/com/epam/pipeline/manager/datastorage/providers/gcp/GSBucketStorageHelper.java

```diff
@@ -69,20 +79,26 @@
 import java.util.TimeZone;
 import java.util.concurrent.TimeUnit;
 import java.util.function.Function;
+import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
 @Slf4j
 @RequiredArgsConstructor
 public class GSBucketStorageHelper {
     private static final String EMPTY_PREFIX = "";
     private static final int REGION_ZONE_LENGTH = -2;
+    private static final String EMPTY_LIFECYCLE_CONTENT = "{}";
+    private static final String LIFECYCLE_CONTENT_TYPE = "application/json";
+    private static final String LIFECYCLE_FIELD = "lifecycle";
+    private static final String GOOGLE_STORAGE_API_URL = "https://www.googleapis.com/storage/";
 
     private static final byte[] EMPTY_FILE_CONTENT = new byte[0];
     private static final Long URL_EXPIRATION = 24 * 60 * 60 * 1000L;
 
     private final MessageHelper messageHelper;
     private final GCPRegion region;
     private final GCPClient gcpClient;
+    private GoogleStorageRestApiClient storageRestApiClient;
```

Comment:
Why do we need as field and not local variable?

Reply:
done